test/e2e/sequential/baseline/default_config_test.go

```
@@ -45,6 +45,11 @@ const (
 )
 
 var _ = ginkgo.Describe("Default configuration tests", ginkgo.Label(util.Shard0), ginkgo.Ordered, func() {
+
+	ginkgo.BeforeAll(func() {
+		util.UpdateKueueConfigurationAndRestart(ctx, k8sClient, defaultKueueCfg, kindClusterName)
+	})
+	
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gci)